src/pybind/mgr/dashboard_v2/frontend/src/app/shared/datatable/table/table.component.ts

```diff
@@ -53,6 +55,12 @@ export class TableComponent implements AfterContentChecked, OnInit, OnChanges {
   // the details page, return false.
   @Input() beforeShowDetails: Function;
 
+  // Auto reload time in ms - per default every 10s
+  @Input() autoReload = 10000;
```

Comment:
Isn't 5000 a better default since it's the value used by all tables?

Reply:
done